<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>options</class>
 <widget class="QDialog" name="options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>change working dirs</string>
  </property>
  <widget class="QPushButton" name="OKButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>100</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>10</y>
     <width>451</width>
     <height>85</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>choose data directory: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="dataEdit"/>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="dataButton">
      <property name="text">
       <string>BROWSE</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>choose list save directory: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="txtEdit"/>
    </item>
    <item row="1" column="2" rowspan="2">
     <widget class="QPushButton" name="listButton">
      <property name="text">
       <string>BROWSE</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" rowspan="2">
     <widget class="QLineEdit" name="pwEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>choose pw save directory: </string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pwButton">
      <property name="text">
       <string>BROWSE</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
